src/Modules/UngradedModules.jl

```diff
@@ -6966,3 +6966,73 @@ function change_base_ring(f::Hecke.Map{DomType, CodType}, M::SubQuo) where {DomT
   map = SubQuoHom(M, MS, gens(MS), f)
   return MS, map
 end
+
```

Comment:
@jankoboehm : 

Could you have a look at this section? I didn't find that functionality, so I just wrote it up. I hope it goes along with your ideas on the design.